### PR TITLE
Add IoT wifi support

### PR DESCRIPTION
**Pull request by adamgross42**
Enum values and README updates

I am surprised that this required the "guest" value on https://github.com/AlexandrErohin/TP-Link-Archer-C6U/blob/main/tplinkrouterc6u/client.py#L57 - but it seems to work still.

Tested with Archer AX3000.

### CODE REVIEW

**Comment by adamgross42** [comment]
I'm not sure how to handle the situation where these IoT fields aren't supported on a device model - there may need to be None-checks in a few places.

**Comment by adamgross42** [comment]
I attempted to rebase on the latest 'main' branch, duplicate commits may have to be squashed.

**Comment by AlexandrErohin** [comment]
@adamgross42 Thanks!
Could you check my comment for making iot_2g_enable bool | None

**Comment by adamgross42** [comment]
Hmm, I don't see any comments in this PR - maybe the review needs to be submitted for me to see them?

**Review by AlexandrErohin — CHANGES_REQUESTED** [review]

File: tplinkrouterc6u/client.py [hunk 1]
```diff
@@ -144,6 +144,8 @@ def _calc_cpu_usage(data: dict) -> float | None:
         status.clients_total = status.wired_total + status.wifi_clients_total + status.guest_clients_total
         status.guest_2g_enable = data.get('guest_2g_enable') == 'on'
         status.guest_5g_enable = data.get('guest_5g_enable') == 'on'
+        status.iot_2g_enable = data.get('iot_2g_enable') == 'on'
```
[the remark this answers is not in the document]
Comment (adamgross42) [reply tplinkrouterc6u/client.py:147]: I can add that in.

Should I also change these two lines to add ` | None`?

https://github.com/AlexandrErohin/TP-Link-Archer-C6U/blob/dccfb505713c1d3ae9f54d4cfa17b5c5eebad5dc/tplinkrouterc6u/dataclass.py#L31-L32

[the remark this answers is not in the document]
Comment (adamgross42) [reply tplinkrouterc6u/client.py:147]: Ah, the typechecker tells me yes, I'll add that too :)

**Comment by adamgross42** [comment]
@AlexandrErohin , what are your thoughts on this?  Are you able to test this locally to confirm the null checks work for a router that doesn't have the IoT network?

**Comment by AlexandrErohin** [comment]
@adamgross42 Hi! Thank you for this PR. Unfortunately, I didnt have time to test it on my router. Give me a week please to check this.

**Comment by AlexandrErohin** [comment]
@adamgross42 I have released it https://pypi.org/project/tplinkrouterc6u/1.3.0/
Thank you!

**Pull request merged**